lldb/cmake/modules/LLDBConfig.cmake

```diff
@@ -8,6 +8,12 @@ set(LLDB_INCLUDE_ROOT "${CMAKE_CURRENT_SOURCE_DIR}/include")
 set(LLDB_SOURCE_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 set(LLDB_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR})
 
+if (LLDB_BUILT_STANDALONE)
+  set(LLDB_OBJ_DIR ${CMAKE_CURRENT_BINARY_DIR})
+else()
+  set(LLDB_OBJ_DIR ${CMAKE_CURRENT_BINARY_DIR}/tools/lldb)
+endif()
```

Comment:
`LLDBConfig` shouldn't know about the standalone build. You should set it to `CMAKE_CURRENT_BINARY_DIR` here and overwrite it in `LLDBStandalone.cmake`.

Reply:
LLDBConfig is included after LLDBStandalone, so that can't work right?

Reply:
It looks like when it's evaluated here it actually points to `tools/lldb` so just setting the variable to CMAKE_CURRENT_BINARY_DIR here seems to do the trick. Unless it's undefined which directory we get based on context?

Reply:
Hmm, thinking about this further that does make sense. CMake will create a subdirectory in your build tree for lldb at `$BUILD_DIR/tools/lldb` and will set `CMAKE_CURRENT_BINARY_DIR` to that subdirectory in that context. For standalone builds, LLDB is its own standalone project, so it gets set to `$BUILD_DIR` in that case. I think you can just set `LLDB_OBJ_DIR` to `${CMAKE_CURRENT_BINARY_DIR}` and call it a day.